lib/screens/login.dart

```diff
@@ -0,0 +1,187 @@
+import 'package:flutter/material.dart';
+// ignore: import_of_legacy_library_into_null_safe
```

Comment:
di to maganda sir,
di kasi supported sound null safety sa package na to(flutter_icons),
since bago pa lang yung project mas maganda kung ayusin na natin yung structure ngayon pa lang parang abandoned na rin yata yung flutter_icons, antagal nang walang updates dun sa repository nila eh https://github.com/flutter-studio/flutter-icons,
also, I think wala na yata yung social media logins, wala na kasi sa wireframes
![asdf](https://user-images.githubusercontent.com/62323308/130411236-5d444d6c-7db1-4bdb-a159-c6e4f2fa9c60.png)

Reply:
yun nga po hindi ko na kasi chineck yung wireframes since akala ko wala ng changes alisin ko nalang po

Reply:
question lang dun sa LoginScreen:
after mai-type ni user yung username niya diba proceed to password? mas okay ba kung dun nalng ilagay yung remember me?

Reply:
dyan muna ser, for now sundin na lang natin yung nasa wireframes, sila na bahala mag edit dun, 
focus lang tayo sa task natin which is coding :+1: